submerge:
  defaults:
    submerge_depth: 1.5
    timeout_time: 10